governator-core/src/main/java/com/netflix/governator/AbstractLifecycleListener.java

```diff
@@ -1,6 +1,7 @@
 package com.netflix.governator;
 
-public class DefaultLifecycleListener implements LifecycleListener {
+
+public abstract class AbstractLifecycleListener implements LifecycleListener {
```

Comment:
Can we make package private to limit our worries of breaking people outside in the future?